README.md

```diff
@@ -5,15 +5,24 @@
 ### Prerequisites
 1. Tomcat 9
 2. Maven
+3. MySQL 8
+4. sef-core
 
-### Building the project
-
-1. Build the project using maven
+### Building the SEF-Core
+- SEF-Core includes the main api's so you need to build it in order to build this project.  
```

Comment:
Sef core includes APIs and the admin dashboard for the academic application.
In order to work with academix-frontend, you need to build sef-core and deploy it on tomcat (core.war and dashboard.war). 
Follow instructions in [sef-core](https://github.com/sef-global/sef-core/blob/master/Readme.md) to build it.

Reply:
Thanks